docs/changelog.md

```diff
@@ -165,6 +165,9 @@ awesomeBar.layout = CustomSuggestionLayout()
 * **browser-toolbar**
   * Fixing bug that allowed text behind the security icon being selectable. [Issue #448](https://github.com/mozilla-mobile/reference-browser/issues/448)
 
+* **feature-downloads**
+  * Introduced a new `onPermissionsDenied` method, which similar to `onPermissionsGranted`, should be invoked in response to permission requests. If the requested permissions were denied the feature will clear the pending download.
+
```

Comment:
Nit: Because of my failure to merge this before the release; this now needs to be moved up to 0.41.0 :)